src/views/Main/Main.tsx

```diff
@@ -75,29 +72,24 @@ export function MainView({ userModel }: MainViewProps) {
       );
     }
     if (userModel?.hasFeature(AppFeature.FOCUSED_GEOMETRY_LAYER)) {
-      import('~features/focused_geometry_layer').then(
-        ({ initFocusedGeometryLayer }) => initFocusedGeometryLayer(),
+      import('~features/focused_geometry_layer').then(({ initFocusedGeometryLayer }) =>
+        initFocusedGeometryLayer(),
       );
     }
-    if (userModel?.hasFeature(AppFeature.REPORTS)) {
```

Comment:
it looks like now we init report independent from AppFeature.REPORTS

`
export function initModes() {
  registerMapMode(modesControlsAtom);
  registerReportsMode(modesControlsAtom);
}`

is it expected?